src/Block/BannerBlock.php

```diff
@@ -52,11 +52,25 @@ public function getCMSFields()
      *
      * @return ArrayData|null
      */
-    public function CallToActionLink()
+    public function getCallToActionLink()
     {
         return $this->decodeLinkData($this->getField('CallToActionLink'));
     }
 
+    /**
+     * Alias of getCallToActionLink()
+     * Originally this was the only function, however it was not being called by PHP
+     * i.e. $bannerBlock->CallToActionLink
+     *
+     * This function is only being kept for semver as removing it would be a breaking change
+     *
```

Comment:
```suggestion
     *
     * @deprecated 5.0.0
```

Reply:
My original solution caused a regression which showed up in a failed behat test.  I've redone my solution.